src/App.test.jsx

```diff
@@ -50,7 +57,7 @@ test("renders Accordion List", () => {
 });
 
 test("renders NavBar", () => {
-  const wrapper = shallow(<NavBar />);
-  const navBarComponent = findByTestAttr(wrapper, "component-navBar");
+  const wrapper = shallow(<Navigation />);
+  const navBarComponent = findByTestAttr(wrapper, "navigation-display");
```

Comment:
🔧 Can you change the variable name from navBar to navigation?

Reply:
Testing for this is being handled in this commit:  d0ac0b0acf20c4dbd5f0a82c6436784a12ee149c